sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/LoadBalancingStrategy.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Messaging.EventHubs.Processor
+{
+    /// <summary>
+    ///   The strategy that an event processor will use to make decisions about
+    ///   partition ownership when performing load balancing to share work with
+    ///   other event processors.
+    /// </summary>
+    ///
+    public enum LoadBalancingStrategy
+    {
+        /// <summary>
+        ///   An event processor will take a measured approach to requesting
+        ///   partition ownership when balancing work with other processors, slowly
+        ///   adding partitions until it reaches its fair share.
```

Comment:
suggestion: `adding partitions until it reaches its fair share.` => `claiming partitions until a stabilized distribution is achieved.`